pipelines/build/regeneration/jdk10_regeneration_pipeline.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+String javaVersion = "jdk10"
```

Comment:
Can we not make all these duplicated files into 1, i.e if regeneration_pipeline.groovy took a runtime parameter to control which are regenerated, would it not make these duplicated files unnecessary?

Reply:
Agreed. Will condense this down

Reply:
Ref https://github.com/AdoptOpenJDK/openjdk-build/pull/1547#discussion_r396441475, the problem is that if all the duplicated files are condensed into one, they will all be ran as one job in jenkins. If its split up, each version will run its `jdkxx_regeneration_pipline` file and not be held up by a different jdk version that is running

Reply:
would using parallel not work here? Same way we use parallel in build_base_file.groovy to create a task for each build job from a single top level job, and they dont block each other, could we not create a parallel task for each regeneration and not block eachother?

Reply:
For the purposes of recording information in public, it's been decided to leave this as is for now and revisit it at a later date